qiskit/quantum_info/operators/symplectic/pauli.py

```diff
@@ -184,27 +183,6 @@ def __init__(self, data=None, x=None, *, z=None, label=None):
             base_z, base_x, base_phase = self._from_scalar_op(data)
         elif isinstance(data, (QuantumCircuit, Instruction)):
             base_z, base_x, base_phase = self._from_circuit(data)
-        elif x is not None:
-            if z is None:
-                # Using old Pauli initialization with positional args instead of kwargs
-                z = data
-            warnings.warn(
-                "Passing 'z' and 'x' arrays separately to 'Pauli' is deprecated as of"
-                " Qiskit Terra 0.17 and will be removed in version 0.23 or later."
-                " Use a tuple instead, such as 'Pauli((z, x[, phase]))'.",
-                DeprecationWarning,
-                stacklevel=2,
-            )
-            base_z, base_x, base_phase = self._from_array(z, x)
-        elif label is not None:
-            warnings.warn(
-                "The 'label' keyword argument of 'Pauli' is deprecated as of"
-                " Qiskit Terra 0.17 and will be removed in version 0.23 or later."
```

Comment:
The release under development right now is 0.22, should we wait until after 0.22 is released to do this because the warning message says 0.23?

Reply:
my bad. I forgot reinstating the deprecation.